src/styles/blocks/catalog.scss

```
@@ -18,8 +18,15 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition: transform 300ms, $smooth-color-transition;
```

Create variables for this & replace them in all cases

```suggestion
    transition: transform $transitionDuration, color $transitionDuration;
```